### DISCUSSION TITLE
WSA 2311 Update

### DISCUSSION THREAD

**Discussion by jaholme** [microsoft/WSA#489] [Announcements]
# What's New

We've shipped an update for Windows Subsystem for Android™ on Windows 11 to the [Windows Subsystem for Android™ Preview Program](https://learn.microsoft.com/en-us/windows/android/wsa/preview-program). This update (2311.40000.3.0) includes improvements to platform reliability and functionality improvements.

## Release Notes

- Platform reliability improvements
- Updated Chromium WebView to version 118

# Giving feedback

If you are having issues with Windows Subsystem for Android™ – please file feedback via Feedback Hub under Apps > Windows Subsystem for Android™. For more information about troubleshooting and submitting feedback: [Troubleshooting and FAQ for mobile apps on Windows](https://support.microsoft.com/en-us/windows/troubleshooting-and-faq-for-mobile-apps-on-windows-d6062afd-98a1-4018-a7c8-6b3b680a2ea5).

Thanks,
Windows Subsystem for Android™ Team

**Comment by hugo-lyppens** [comment]
Is local network SSDP discovery working again like it used to in WSA 2305?

**Comment by bubbleguuum** [comment]
> Is local network SSDP discovery working again like it used to in WSA 2305?

Sadly, it doesn't.

Please MS team, make SSDP and mDNS UDP multicast discovery work again on the local network ! it worked in WSA 2305 and prior.
